h2o-core/src/main/java/water/util/ArrayUtils.java

```diff
@@ -1548,8 +1548,7 @@ public static void sort(final int[] idxs, final double[] values, int cutoff, int
       Arrays.sort(d, new Comparator<Integer>() {
         @Override
         public int compare(Integer x, Integer y) {
-          return values[x]*increasing < values[y]*increasing ? -1 : 
-                  (values[x]*increasing > values[y]*increasing ? 1 : 0);
+          return Double.compare(values[x] * increasing, values[y] * increasing);
```

Comment:
technically this is changing the behavior - because Double.compare handles also special cases like NaN, -NaN and -0

Reply:
I would probably leave as-is in this PR, unless you have a reason to change

Reply:
Aaa ok, I'll remove the change, the change is only because of the code style

Reply:
Removed